### PR TITLE
[Docs] Add v1.7 updates to change log

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: docs/change-log/index.md [hunk 1]
```diff
@@ -11,6 +14,156 @@
...
+### Alerts
+| ID    |Description                                                                 |
+|-------|----------------------------------------------------------------------------|
+|ML-5287|You can now configure {ref}`alerts <alerts>` for model monitoring and other possible problem situations. TechPreview. |
+
+
+### Projects
+| ID    |Description                                                                 |
+|-------|----------------------------------------------------------------------------|
+|ML-2585|When deleting a project, you can also [delete a single artifact](../store/artifacts.html#deleting-artifacts).|
```
Comment (moranbental) [inline docs/change-log/index.md]: I think ML-2585 should be under the artifact section, not the project section
```suggestion
|ML-2585|When deleting a artifact, you can also [delete a single artifact content](../store/artifacts.html#deleting-artifacts).|
```

File: docs/concepts/notifications.md
Comment (roei3000b) [inline docs/concepts/notifications.md:1]: @jillnogold , where is my change on [this pr](https://github.com/mlrun/mlrun/pull/6532/files#diff-5788acdc06e77df7d2830bfdc2fbc9a2788a8b2c082ec16fd1a2e0fd731e0cf6)?

File: docs/change-log/index.md [hunk 2]
```diff
@@ -11,6 +14,156 @@
...
+|ML-6065|Fixed serving graphs when working with Kafka.|
+|ML-6194|When running a remote workflow, the client side does not need to contain workflow local files.|
+|ML-6068|Feature-store Redis-target is now created by default with the project/feature-set in the path.|
+|ML-6249|Reduction in the time that feature sets with a large number of entities take to query (and no longer timeout).|
+|ML-6253|The `project.build_function()`now passes the `requirements_file parameter` when the  engine is not `kfp`.|
+|ML-6602/6556|You can now specify a UID when running `artifacts.get_model`.|
+|ML-6800|Resolved spiking of CPU and memory of mlrun-api service.|
+|ML-6885|Webhook notification now contains run logs when `override_body=True`|
+|ML-6991|UI: Now shows KFP pod errors.|
+|ML-7103|UI: New option to show untagged functions in the Functions page (disabled by default). Added time filters to the fFunctions page, by default shows only functions that were modified in the last week.|
```
Comment (moranbental) [inline docs/change-log/index.md]:
```suggestion
|ML-7103|UI: New option to show untagged functions in the Functions page (disabled by default). Added time filters to the Functions page, by default shows only functions that were modified in the last week.|
```

**Review by moranbental — CHANGES_REQUESTED** [review]

File: docs/change-log/index.md [hunk 3]
```diff
@@ -11,6 +14,156 @@
...
+|ML-7256|Deleting artifact by UID deletes only the artifact with the given UID.|
+|ML-7290|Nuclio functions deployed by MLRun: At the Nuclio service level, node selectors are added by Nuclio. In cases where there is overlap between the node selectors specified at the MLRun service level and those applied by Nuclio, the node selectors from the MLRun side take precedence to prevent any conflicts. See [node selection](../runtimes/configuring-job-resources.html#node-selection).|
+|ML-7335|Serving functions now support up to 4500 models and the size of the serving specification to 1 MB.|
+|ML-7358|MLRun client now validates certificates by default.|
+|ML-7367|Updated OS packages for MLRun UI.|
+|ML-7416|An error message is displayed upon workflow failure in the "Monitor workflows" tab.|
+|ML-7599|Fixed mlrun-api vulnerabilities.|
+|ML-7568/7915| The SDK does not inform of invalid node selector combinations when running a function, but the pod remains stuck in the Pending state. |See [Preventing and resolving conflicts](../runtimes/configuring-job-resources.html#preventing-and-resolving-conflicts). v1.7.0|
```
Comment (alonmr) [inline docs/change-log/index.md]: This sounds like an open issue

File: docs/change-log/index.md [hunk 4]
```diff
@@ -11,6 +14,156 @@
...
+|ML-7613|The pipeline SDK output now displays the logs in addition to the graph animation, and it also raises a warning when a client uses notifications with `engine=kfp`.|
+|ML-7655|The "filter by label" in the Models page is now case insensitive.|
+|ML-7673|Improved performance of `db.list_pipelines()` with project filter in environments with a large number of pipelines,|
```
Comment (alonmr) [inline docs/change-log/index.md]:
```suggestion
|ML-7673|Improved performance of `db.list_pipelines()` with project filter in environments with a large number of pipelines.|
```

File: docs/change-log/index.md [hunk 5]
```diff
@@ -11,6 +14,156 @@
...
+|ML-7706|Fixed the skip `sync_functions` when scheduling workflows or running remote workflows.|
```
Comment (alonmr) [inline docs/change-log/index.md]:
```suggestion
|ML-7706|Change `sync_functions` to be "best effort" when scheduling workflows or running remote workflows.|
```

**Review by alonmr — COMMENTED** [review]
Small comments from me

File: docs/change-log/index.md [hunk 6]
```diff
@@ -11,6 +14,156 @@
...
+| ID    |Description                                                                 |
+|-------|----------------------------------------------------------------------------|
+|ML-2585|When deleting a project, you can also [delete a single artifact](../store/artifacts.html#deleting-artifacts).|
+|ML-3874|Node selector can now be [defined in the project spec](../runtimes/configuring-job-resources.html#node-selection).|
+
+### Runtimes
+| ID    |Description                                                                 |
+|-------|----------------------------------------------------------------------------|
+|ML-2652|Supports [creating an API gateway using the MLRun SDK](../concepts/nuclio-real-time-functions.html#api-gateway).|
+|ML-4601|New {ref}`application runtime <application>` where you can run provide an image (for example a web-app) that runs as a deployment. |
```
Comment (TomerShor) [inline docs/change-log/index.md]:
```suggestion
|ML-4601|New {ref}`Application runtime <application>` where you can provide an image (a web-app, for example) that runs as a kubernetes deployment. |
```

File: docs/runtimes/configuring-job-resources.md [hunk 7]
```diff
@@ -365,21 +368,21 @@ The zone label from the project level is completely removed, and the resulting c
...
 ```
-### Preventing conflicts
+### Preventing and resolving conflicts
 
 If your function run is stuck with the status `pending`, it's possible that the "specified" node selector does not exist. There are three 
 levels of node selectors in MLRun: function, project, and service. At runtime, the system combines these selectors and applies the resolved 
 configuration to the pod.
 
-How to Investigate:
+**How to Investigate:
```
Comment (TomerShor) [inline docs/runtimes/configuring-job-resources.md]:
```suggestion
**How to Investigate:**
```

**Review by TomerShor — CHANGES_REQUESTED** [review]

File: docs/change-log/index.md [hunk 8]
```diff
@@ -11,6 +14,160 @@
...
+|ML-7135|Fixed upgrading to mlrun if the `spec.build` was saved incorrectly.|
+|ML-7162|UI: Erroneous "“No data matches…” messages no longer appear.|
+|ML-7192|Improved the liveness and readiness probes responsiveness.|
+|ML-7202|The run state now indicates if a pod failed on OOM.|
+|ML-7226|Exception now raised when using use KFP engine with `pass schedule` argument, which is not supported with the KFP engine.|
+|ML-7256|Deleting artifact by UID deletes only the artifact with the given UID.|
+|ML-7290|Nuclio functions deployed by MLRun: At the Nuclio service level, node selectors are added by Nuclio. In cases where there is overlap between the node selectors specified at the MLRun service level and those applied by Nuclio, the node selectors from the MLRun side take precedence to prevent any conflicts. See [node selection](../runtimes/configuring-job-resources.html#node-selection).|
+|ML-7335|Serving functions now support up to 4500 models and the size of the serving specification to 1 MB.|
+|ML-7358|MLRun client now validates certificates by default.|
```
Comment (alonmr) [inline docs/change-log/index.md]:
```suggestion
|ML-7358|MLRun client now validates SSL certificates by default.|
```

File: docs/change-log/index.md [hunk 9]
```diff
@@ -906,77 +1070,71 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|NA|The feature store does not support schema evolution and does not have schema enforcement.| NA| v1.2.1    |
+|ML-3526|Aggregation column order is not always respected (storey engine).| NA | v1.3.0|
+|ML-3626|The "Save and ingest" option is disabled for a scheduled feature set. |NA | v1.3.0|
+|ML-3627|The feature store allows ingestion of string type for the timestamp key resulting in errors when trying to query the offline store with time filtration.|Use only timestamp type.| v1.2.1    |
+|ML-3636|`get_online_feature_service` from Redis target returns truncated values. | NA | v1.3.0|
+|ML-3640|When running a remote function/workflow, the `context` global parameter is not automatically injected.| Use `get_or_create_ctx`| v1.3.0    |
```
Comment (alonmr) [inline docs/change-log/index.md]: The ticket here is closed I think we can remove this one

File: docs/change-log/index.md [hunk 10]
```diff
@@ -906,77 +1070,71 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|NL-4685|When using columns with type "float" as feature set entities, they are saved inconsistently to key-value stores by different engines.| Do not use columns with type float as feature set entities. |v1.5.0 |
+|ML-4698|Parameters that are passed to a workflow are limited to 10000 chars.<!-- also mentioned in /concepts/submitting-tasks-jobs-to-functions.html -->| NA, external Kubeflow limitation. | v1.5.0 |
+|ML-4725|ML functions show as if they are in the  "Creating" status, although they were created and used.|NA|v1.4.1|
+|ML-4740|When running function `batch_inference_v2` from the SDK, the `ingest()` function accepts 3 parameters as Data-item or other types: `dataset`, `model_path` and `model_endpoint_sample_set`. If you provided these parameters as non Data-items and later on you want to rerun this function from the UI, you need to provide these parameters as Data-item.|Prepare suitable Data-item and provide it to the batch-rerun UI.| v1.5.0    |
+|ML-4769|After deleting a project, data is still present in the Artifacts and Executions of pipelines UI.  | NA | v1.4.0 |
+|ML-4857|Local runs can be aborted in the UI, though the actual execution continues.|NA | v1.5.0 |
```
Comment (alonmr) [inline docs/change-log/index.md]: This is also closed

File: docs/change-log/index.md [hunk 11]
```diff
@@ -906,77 +1070,71 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|NL-4881|Kubeflow pipelines parallelism parameter in dsl.ParallelFor() does not work (external dependency). |NA| v1.4.1|
```
Comment (alonmr) [inline docs/change-log/index.md]:
```suggestion
|ML-4881|Kubeflow pipelines parallelism parameter in dsl.ParallelFor() does not work (external dependency). |NA| v1.4.1|
```

**Review by alonmr — APPROVED** [review]
LGTM

**Review by TomerShor — APPROVED** [review]

**Pull request merged**